SWLOR.Game.Server/Feature/MigrationDefinition/ServerMigration/_15_ShipRebalance.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
```

Comment:
![image](https://github.com/user-attachments/assets/60ca459e-b08d-4a41-a97f-bc23c0d90535)

Reply:
The logic in this file should be put into ServerMigrationBase. Put it under a new function, then call that in this migration. Also update migration 14 so we aren't duplicating code